lib/srv/ctx.go

```diff
@@ -264,12 +264,29 @@ func NewServerContext(srv Server, conn *ssh.ServerConn, identityContext Identity
 	})
 
 	if !ctx.disconnectExpiredCert.IsZero() || ctx.clientIdleTimeout != 0 {
-		go ctx.periodicCheckDisconnect()
```

Comment:
I don't think anyone else is calling this function, so you can remove the function as well.